noah_frontend/src/components/donate.js

```diff
@@ -66,6 +66,11 @@ class DonateView extends React.Component {
     addItem = (e) => {
         e.preventDefault()
         const name = prompt("Enter name of the Item")
+        const nameRegex = /^[a-zA-Z][-_ a-zA-Z0-9]+$/	
```

Comment:
`A_` is a valid name is it expected?

Reply:
There was a problem with the old regex, it was not supporting *space*. So I just updated it with space, -, and '_'. Will remove it if it's invalid

Reply:
It is ok  if A_ is expected to be a valid name or if we want to give flexibility to the validations